src/contact-page/styles.css

```diff
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  border: none;
+  background: none;
+} 
+.main-container{
+  width: 100%;
+  height: 1000px;
+  display: flex;
+  flex-direction: row;
+  border: solid 1px pink;
+  flex-grow: 1;
+}
+.left-box{
+  width: 450px;
+  height: 1000px;
+  background-color: #EDAE9B;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  
+  direction: ltr;
+  justify-content: flex-start;
+  align-items: flex-end;
+  font-size: 100px;
+  font-family: 'Lato', sans-serif;
+}
+.right-box{
+  width: 60%;
+  height: 1000px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  background-color: rgb(182, 182, 185);
+  font-family: 'Amatic SC', cursive;
+  box-shadow: 4px 0px 17px 0px inset;
```

Comment:
consider adding some padding to this to get the text/inputs away from the edge. `padding: 0 40px;`